v19.2/drop-table.md

```diff
@@ -62,52 +65,82 @@ DROP TABLE
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW TABLES FROM bank;
+> SHOW TABLES FROM movr;
 ~~~
 
 ~~~
-+--------------------+
-| table_name         |
-+--------------------+
-| accounts           |
-| user_accounts_view |
-+--------------------+
-(2 rows)
+          table_name
++----------------------------+
+  rides
+  user_promo_codes
+  users
+  vehicle_location_histories
+  vehicles
+(5 rows)
 ~~~
 
 ### Remove a table and dependent objects with `CASCADE`
 
-In this example, a view depends on the table being dropped. Therefore, it's only possible to drop the table while simultaneously dropping the dependent view using `CASCADE`.
+In this example, a foreign key from a different table references the table being dropped. Therefore, it's only possible to drop the table while simultaneously dropping the dependent foreign key constraint using `CASCADE`.
```

Comment:
It might be nice to make the first mention of "foreign key" into a link to the Foreign Key Constraint doc.

Reply:
Done.